Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -229,6 +229,9 @@ document title. This way you can link to a documents start page by:
 
     See :ref:`TYPO3 Explained <t3coreapi:start>`.
 
+..  hint::
+    Refer to https://typo3.org/project/licenses if you have questions about
```

Comment:
Attribute as link?

Reply:
URIs are automatically detected, you only need to do that if you want to give a different title